plugins/webp-uploads/helper.php

```diff
@@ -16,20 +16,24 @@
  * for example an image/jpeg can be converted into an image/webp.
  *
  * @since 1.0.0
+ * @since n.e.x.t Added support for AVIF.
  *
  * @return array<string, array<string>> An array of valid mime types, where the key is the mime type and the value is the extension type.
  */
 function webp_uploads_get_upload_image_mime_transforms() {
+
+	// Check the selected output format.
+	$output_format = get_option( 'perflab_generate_avif_and_webp', wp_image_editor_supports( array( 'mime_type' => 'image/avif' ) ) ? 'avif' : 'webp' );
```

Comment:
Since the `wp_image_editor_supports( array( 'mime_type' => 'image/avif' )` code is appearing in multiple places in this PR, should there be a helper function like `webp_uploads_supports_avif()`?

Reply:
Also, the `perflab_generate_avif_and_webp` option here only exists if AVIF support is present. Otherwise, the option will never exist which will result in an additional failing DB query for each request unless a persistent object cache is present, right? Should the same condition then be used? Like:

```suggestion
	$output_format = wp_image_editor_supports( array( 'mime_type' => 'image/avif' ) ) ? get_option( 'perflab_generate_avif_and_webp' ) : 'webp';
```